### DISCUSSION TITLE
Temporarily Avoid Talkgroup

### DISCUSSION THREAD

**Discussion by tadscottsmith** [chuot/rdio-scanner#218] [Ideas]
I know button space is at a premium, but I wonder how people feel about a "temporary lockout" button. When things get really busy, I need to limit what I listen to, but I sometimes forget to re-enable the talk groups. I think it would be super beneficial to have a temporary (60 minute? 2 hour?) lockout button.

**Comment by chuot** [comment]
I really like the idea and I would use it frequently!

Now, indeed button space is premium and that's why I'm thinking of changing the way the AVOID button works. Say that instead of just toggling between avoided state and not avoided state, I could cycle the states between AVOID 30M -> AVOID 1H -> AVOID 2H -> AVOID -> NORMAL (not avoided).

The only issue I see with that is when the QUEUE is loaded, as soon as you press on the AVOID button, it clears the QUEUE for the current playing talkgroup and quickly plays the next call in the QUEUE. So no time to cycle between the states because the call is already gone. Then, to cycle the states you would have to PAUSE, then AVOID once for AVOID 30M, twice for AVOID 1H, then PAUSE to restart playing. Again, this is only when the QUEUE is loaded. If it ain't, then no need to PAUSE, just cycle the states by pressing AVOID a couple of times to reach to desired one.

Your thoughts?

**Reply by tadscottsmith** [reply]
I almost always have a play queue, so I've gotten into the habit of hitting PAUSE, AVOID, PAUSE to lock them out for 30 minutes, and as far as I can tell, it seems to be working great. Thanks for implementing this!
